### PR TITLE
Use approxEq to judge double compare in math-cbrt.js

### PR DESCRIPTION
**Pull request by lygstate**
JerryScript-DCO-1.0-Signed-off-by: Yonggang Luo luoyonggang@gmail.com

### CODE REVIEW

File: tests/jerry/es.next/math-cbrt.js [hunk 1]
```diff
@@ -20,6 +20,11 @@ function isSameZero (x, y)
   return x === 0 && (1 / x) === (1 / y);
 }
 
+function isSameDouble (x, y)
```
Comment (akosthekiss) [inline tests/jerry/es.next/math-cbrt.js:23]: I suggest to use a different name for this function. Perhaps `approxEq` is the shortest name that could be used. It uses "accepted" abbreviations, describes what comparison it performs, and does not conflict with English grammar. (`isSameZero` isn't really a good pattern to follow, unfortunately.)

**Review by akosthekiss — CHANGES_REQUESTED** [review]
Will be OK technically after a terminology fix.

File: tests/jerry/es.next/math-cbrt.js (hunk 1 above)
Comment (lygstate) [reply tests/jerry/es.next/math-cbrt.js:23]: To follow function

function isSameZero (x, y)
{
  return x === 0 && (1 / x) === (1 / y);
}

**Review by akosthekiss — APPROVED** [review]
LGTM

**Review by akosthekiss — CHANGES_REQUESTED** [review]
Ah, late hit. Missed the commit message. Please update that.

**Review by akosthekiss — APPROVED** [review]
LGTM

File: tests/jerry/es.next/math-cbrt.js [hunk 2]
```diff
@@ -20,6 +20,11 @@ function isSameZero (x, y)
   return x === 0 && (1 / x) === (1 / y);
 }
 
+function approxEq (x, y)
+{
+  return Math.abs(x - y) < 1e-15
```
Comment (rerobika) [inline tests/jerry/es.next/math-cbrt.js:25]: How about using `Number.EPSILON` instead of a handcrafted one?

**Review by rerobika — CHANGES_REQUESTED** [review]

File: tests/jerry/es.next/math-cbrt.js (hunk 2 above)
Comment (lygstate) [reply tests/jerry/es.next/math-cbrt.js:25]: Not sure, 1e-16 failed before.
Let's me have a try.
The EPSILON property has a value of approximately 2.2204460492503130808472633361816E-16

Comment (lygstate) [reply tests/jerry/es.next/math-cbrt.js]: [ 212/1145] FAIL (-6): tests/jerry/es.next/math-cbrt.js
================================================
Script Error: assertion failed

================================================

[summary] build/tests/jerry_tests-es.next-debug/local/bin/jerry tests/jerry

TOTAL: 1145
PASS: 1144
FAIL: 1

Success: 99%

Not working, how about Number.EPSILON * 10? or other number

**Review by rerobika — APPROVED** [review]
LGTM

**Pull request merged**